Resolve #282.

gspinaci opened the pull request:
Environment SecretResolver is working now. 

I tested it only for development mode. In the next days I will be test it in production mode (Docker).
Then, i will create the documentation

doldman commented:
Thanks for this - Will be reviewed by Kartography in the next few days

gspinaci commented:
Unfortunately, the build is failing because of https://github.com/researchspace/researchspace/pull/283

gspinaci commented:
#283 has been marged into master. Now the build is working. 

I am testing this code on docker

gspinaci commented:
The code is working correctly passing secrets as JAVA OPTS running a Docker instance. 

@doldman I would like to create a documentation page on how to set these parameters. Can you please point me to the right folder where to create a documentation page to be integrated into https://documentation.researchspace.org/?

dianat commented:
Please upload here:
https://drive.google.com/drive/folders/1gNF44hd8k4l9YIZT-CmxNc0qR6xxtvdr

Thank you.

gspinaci commented:
I added the documentation file. It is called Environment SecretResolver. I have also added the change's reason, description, and details.

Please, let me know if everything is okay,
Thanks.
